techmatters/terraso-mobile-client
# fix: permissions for site note editing

## Description

Restrict the site note edit button to only the note author or the site's owners/project's managers. Additionally, only the note author can submit edits to the note.

### Checklist

### Related Issues
#709

### Verification steps

As a project contributor, create a note for a site. Verify that the edit button is not visible to either a viewer nor a different contributor. Verify that the edit button is visible to a manager. Open the site note edit screen and verify that only the note author can edit the text, but that project managers can still delete the note.
